server/portal/apps/users/views.py

```diff
@@ -146,6 +154,46 @@ def get(self, request, username):
         return JsonResponse({username: user_data})
 
 
+@method_decorator(login_required, name='dispatch')
+class TasUsersView(BaseApiView):
+
+    def get(self, request):
```

Comment:
We should probably add some docstrings throughout this file
```suggestion
class TasUsersView(BaseApiView):
    """SOAP search endpoint for TAS users
    """

    def get(self, request):
```